crates/sequencer/src/listen_mode.rs

```diff
@@ -100,6 +105,12 @@ where
                 |shutdown_signal| async move { self.l1_syncer.run(shutdown_signal).await },
             );
 
+        self.task_executor
+            .spawn_critical_with_graceful_shutdown_signal(
```

Comment:
Don't use spawn critical... this is just for the sequencer runner. 
```suggestion
            .spawn_with_graceful_shutdown_signal(
```